src/commonMain/kotlin/io/github/kevincianfarini/alchemist/internal/SaturatingLong.kt

```diff
@@ -153,3 +160,8 @@ internal value class SaturatingLong(val rawValue: Long) {
 internal inline val Long.saturated get() = SaturatingLong(this)
 internal inline val POSITIVE_INFINITY get() = SaturatingLong(Long.MAX_VALUE)
 internal inline val NEGATIVE_INFINITY get() = SaturatingLong(Long.MIN_VALUE)
+
+// TODO: Supply a ProGuard rule to keep this method so it's not inlined
```

Comment:
Easiest is to put them in `src/jvmMain/resources/META-INF/proguard/alchemist.pro`. This should cover both JVM usage (e.g., Compose Desktop) and Android.

Reply:
https://github.com/kevincianfarini/alchemist/issues/38